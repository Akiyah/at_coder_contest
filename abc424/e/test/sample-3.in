11
3 4 1
40 20 30
3 4 2
40 20 30
3 4 3
40 20 30
3 4 4
40 20 30
3 4 5
40 20 30
3 4 6
40 20 30
3 4 7
40 20 30
3 1 1
40 20 30
3 1 2
40 20 30
3 1 3
40 20 30
3 1 4
40 20 30
