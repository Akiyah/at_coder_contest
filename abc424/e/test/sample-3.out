20.00000000000000000000
15.00000000000000000000
15.00000000000000000000
10.00000000000000000000
10.00000000000000000000
10.00000000000000000000
10.00000000000000000000
30.00000000000000000000
20.00000000000000000000
20.00000000000000000000
20.00000000000000000000
